db/init_db.js

```diff
@@ -447,6 +447,78 @@ async function populateInitialData() {
         price: "$9",
         category: "cheese"
       },
+      {
+        productId: 37,
+        name: "Extra Mature Real Yorkshire Wensleydalee",
```

Comment:
Please change Wensleydalee to **Wensleydale** unless this was intentional then ignore.

Reply:
Pushing through for code review, will resolve later tonight.